rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapStorageProviderImpl.java

```diff
@@ -24,7 +24,7 @@ public Optional<Coin> getLockingCap(ActivationConfig.ForBlock activations) {
             if (lockingCap == null) {
                 initializeLockingCap();
             }
-            return Optional.of(lockingCap);
+            return Optional.ofNullable(lockingCap);
```

Comment:
Could this value still be null after calling `initializeLockingCap`?

Reply:
Yes, when there's no previous value in the storage.

Reply:
It's necessary to run the tests correctly.

Reply:
ohh okey, no worries tho